index.html

```diff
@@ -6,36 +6,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./assets/css/styles.css">
-    <title>First Project</title>
+    <title>Javier Martinez's Portfolio</title>
 </head>
 
 <body>
     <header>
-        <div class="topHeader">
-            <div class="logo">
-                <a href="#" rel="noopener" target="_blank">
-                    My Logo
-                </a>
-            </div>
-            <nav>
-                <ul class="links-index">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                </ul>
-                <div id="breadcrumb">
-                    <a href="#" rel="noopener" target="_blank">
-                        <img src="./assets/images/breadcrumb.png" alt="">
-                    </a>
-                </div>
-            </nav>
+        <div class="logo">
+          <a href="#">My Logo</a>
         </div>
+  
+        <nav>
+          <div class="hamburger">
+            <img src="./assets/images/breadcrumb.png" alt="" />
+          </div>
+          <div class="main-menu">
+            <span class="close">X</span>
+              <a class="links" id="Portfolio" href="#portfolio">Portfolio</a>
+              <a class="links" id="about" href="#about">About</a>
+              <a class="links" id="contact" href="#form-section">Contact</a>
```

Comment:
- [ ] This is actually traversing as appropriate, 😃. But the issue is that "Clicking on any of the links not just the X-Icon should also make the mobile menu disappear.😃. Like it is apparent up there ☝️